src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/AnnotationArchiveRecordMapperImpl.java

```diff
@@ -306,7 +306,15 @@ private UploadNewFileRecordMetadata createUploadNewFileRecordMetadata(Annotation
         UploadNewFileRecordMetadata uploadNewFileRecordMetadata = new UploadNewFileRecordMetadata();
         uploadNewFileRecordMetadata.setPublisher(armDataManagementConfiguration.getPublisher());
         uploadNewFileRecordMetadata.setDzFilename(rawFilename);
-        uploadNewFileRecordMetadata.setFileTag(annotationDocument.getFileType());
+        uploadNewFileRecordMetadata.setFileTag(getFileTag(annotationDocument.getFileName()));
```

Comment:
I think you could use [FilenameUtils.getExtension](https://github.com/search?q=repo%3Ahmcts%2Fdarts-api%20FilenameUtils.getExtension&type=code) here instead if you want to grab the extension?